src/test/resources/additionalfolders/pom.xml

```diff
@@ -21,7 +21,7 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>com.coveo</groupId>
+                <groupId>com.spotify</groupId>
                 <artifactId>fmt-maven-plugin</artifactId>
                 <version>2.12</version>

```

Comment:
interesting, there's a bunch of them that seems outdated

Reply:
Yeah, they are part of the tests, seemed to work either way, but I can doublecheck.

Reply:
I attached a breakpoint to one of the classes, and it seems to work: the breakpoint is hit when running the tests :+1: